### ISSUE TITLE
Add more sources

### ISSUE THREAD

**Issue by Archeix7** [alexmercerind/audire#18]
Please add other sources such as Tidal, Deezer, SoundCloud, Amazon Music, Qobuz and when clamping the button with the source to copy the link

**Comment by sonalder-darlene** [comment]
I agree that having Tidal, Deezer and the other would be really great as Spotify and YouTube don't pay well the artists I tend to avoid these plateforms

**Comment by alexmercerind** [comment]
I decided to go with a new layout, which uses system's default search. We didn't have direct links anyway, the best way is to not link any app at all. It is part of the latest release: [v1.2-3](https://github.com/alexmercerind/audire/releases/tag/v1.2-3).

<img src="https://github.com/user-attachments/assets/24af5d64-02e8-4784-9267-35ad469836ea" height="600">

**Comment by roke-julian-lockhart** [comment]
@alexmercerind, as in, it now invokes `com.android.intentresolver/.ChooserActivityLauncher`? [^1] [^2]

[^2]: [`issues/15#issuecomment-3476428874`](https://github.com/alexmercerind/audire/issues/15#issuecomment-3476428874)

[^1]: [`issues/29#issuecomment-3476430515`](https://github.com/alexmercerind/audire/issues/29#issuecomment-3476430515)

**Comment by alexmercerind** [comment]
Yess.
